src/contracts/minter/facets/DepositWithdrawFacet.sol

```diff
@@ -87,4 +87,81 @@ contract DepositWithdrawFacet is DiamondModifiers, MinterModifiers, IDepositWith
         IERC20Upgradeable(_collateralAsset).safeTransfer(_account, _withdrawAmount);
         emit MinterEvent.CollateralWithdrawn(_account, _collateralAsset, _withdrawAmount);
     }
+
+    /**
+     * @notice Withdraws sender's collateral from the protocol before checking minimum collateral ratio.
+     * @dev Executes post-withdraw-callback triggering onUncheckedCollateralWithdraw on the caller
+     * @dev Requires that the post-withdraw-callback collateral value does not violate minimum collateral requirement.
+     * @param _account The address to withdraw assets for.
+     * @param _collateralAsset The address of the collateral asset.
+     * @param _withdrawAmount The amount of the collateral asset to withdraw.
+     * @param _depositedCollateralAssetIndex The index of the collateral asset in the sender's deposited collateral
+     * assets array. Only needed if withdrawing the entire deposit of a particular collateral asset.
+     */
+    function withdrawCollateralUnchecked(
+        address _account,
+        address _collateralAsset,
+        uint256 _withdrawAmount,
+        uint256 _depositedCollateralAssetIndex
+    )
+        external
+        nonReentrant
+        collateralAssetExists(_collateralAsset)
+        onlyRoleIf(_account != Meta.msgSender(), Role.MANAGER)
+    {
+        if (ms().safetyStateSet) {
+            ensureNotPaused(_collateralAsset, Action.Withdraw);
+        }
+
+        uint256 collateralDeposits = ms().getCollateralDeposits(_account, _collateralAsset);
+        _withdrawAmount = (_withdrawAmount > collateralDeposits ? collateralDeposits : _withdrawAmount);
+
+        // perform unchecked withdrawal
+        ms().recordCollateralWithdrawal(
+            _account,
+            _collateralAsset,
+            _withdrawAmount,
+            collateralDeposits,
+            _depositedCollateralAssetIndex
+        );
+
+        // transfer the withdrawn asset to the caller
+        IERC20Upgradeable(_collateralAsset).safeTransfer(msg.sender, _withdrawAmount);
+
+        // execute the callback on the caller
+        _postUncheckedCollateralWithdrawal(_account, _collateralAsset, _withdrawAmount, _depositedCollateralAssetIndex);
+
+        /*
+         Perform the MCR check after the callback has been executed
+         Ensures accountCollateralValue remains over accountMinColateralValueAtRatio(MCR)
+        */
+        ms().verifyAccountCollateral(_account, _collateralAsset, _withdrawAmount);
```

Comment:
`_withdrawAmount` should be 0 here as we already reduce deposits in `recordCollateralWithdrawal`